docs/python/debugging.md

```diff
@@ -81,7 +103,7 @@ Specifies arguments to pass to the python program, for example:
 
 ### `stopOnEntry`
 
-When set to true, breaks the debugger at the first line of the program being debugged. Setting to false runs the program to the first breakpoint.
+When omitted or set to true, breaks the debugger at the first line of the program being debugged. Setting to false runs the program to the first breakpoint.
```

Comment:
This is not true anymore. The value must be set to true. The current default value is `false`.

Reply:
I'll fix.